scripts/update_homebrew.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# update_homebrew.sh uses `brew bump-formula-pr` to update the formula for the Pulumi CLI and SDKs
+set -o nounset
+set -o errexit
+set -o pipefail
+readonly ROOT=$(dirname "${0}")/..
+
+if [[ "${TRAVIS:-}" != "true" ]]; then
+    echo "error: this script should be run from within Travis"
+    exit 1
+fi
+
+if [[ -z "${PULUMI_BOT_GITHUB_API_TOKEN:-}" ]]; then
+    echo "error: PULUMI_BOT_GITHUB_API_TOKEN must be set"
+    exit 1
+fi
+
+if ! echo "${TRAVIS_TAG:-}" | grep -q -e "^v[0-9]\+\.[0-9]\+\.[0-9]\+$"; then
```

Comment:
Do we have a primary source on this regexp that documents why this is correct?  Could we add a comment that links to that?